datasource/src/test/scala/quasar/physical/s3/SecureS3DatasourceSpec.scala

```diff
@@ -38,22 +38,31 @@ import SecureS3DatasourceSpec._
 final class SecureS3DatasourceSpec extends S3DatasourceSpec {
   override val testBucket = Uri.uri("https://s3.amazonaws.com/slamdata-private-test")
 
-  // FIXME: close the file once we update to cats-effect 1.0.0 and
-  // Bracket is available
   override val credentials: IO[Option[S3Credentials]] = {
-    val file = Sync[IO].catchNonFatal(new File("testCredentials.json"))
-    val msg = "Failed to read testCredentials.json"
+    val read = IO.delay({
```

Comment:
Replace this with `IO {`. The `delay` method is redundant and a waste of five characters. Also no need for the parens either way.